### ISSUE TITLE
\input webmac failes

### ISSUE THREAD

**Issue by razvanm** [ds26gte/tex2page#1]
An `\input webmac` is causing tex2page to throw an error.

Input `magstep.tex` file:

```
\input webmac
\bye
```

The error:

```
bash-3.2$ ../tex2page magstep.tex 
This is TeX2page, Version 20150602 (Racket 6.2, :unix)
(magstep.tex (/opt/local/share/texmf-texlive/tex/plain/base/webmac.tex
! Missing number.
/opt/local/share/texmf-texlive/tex/plain/base/webmac.tex:8: error
l.8 get-number failed.
Likely error context: /opt/local/share/texmf-texlive/tex/plain/base/webmac.tex, lines 6-10:
 /__________________________...
 | \font\eightrm=cmr8 \let\sc=\eightrm % NOT a caps-and-small-caps font!
 | \let\mainfont=\tenrm
 | \font\titlefont=cmr7 scaled\magstep4 % title on the contents page
 | \font\ttitlefont=cmtt10 scaled\magstep2 % typewriter type in title
 | \font\tentex=cmtex10 % TeX extended character set (used in strings)
 |__________________________...
/
Output written on magstep.html (1 page).
Transcript written on magstep.hlog.
Type e to edit file at point of error; x to quit.
? q
TeX2page fatal error
  context...:
   [...]/local/src/tex2page/bug/../tex2page:8324:6: loop
   [...]/local/src/tex2page/bug/../tex2page:8379:2: tex2page-file
   [...]/local/src/tex2page/bug/../tex2page:8324:6: loop
   [...]/local/src/tex2page/bug/../tex2page:8379:2: tex2page-file
bash-3.2$
```

**Comment by ds26gte** [comment]
Please confirm fix in a22bc1d12e06123ae28ae825fa7df0939e564286

**Comment by razvanm** [comment]
The fix works. Thanks!
